### PR TITLE
Revise "Supporting simultaneous multiple transports" proposal

### PR DESCRIPTION
**Pull request by shoamano83**
This proposal aims to support multiple transports between Core and Proxy.

### CODE REVIEW

**Comment by shoamano83** [comment]
Hi @theresalech , this PR is to update "SDL 0141 Supporting simultaneous multiple transports" proposal. This is ready for review.
Thanks.

**Comment by shoamano83** [comment]
Hi @theresalech , I received some comments internally after I pushed commit d417e5c. So please kindly review the latest version.
Thanks.

File: proposals/0141-multiple-transports.md [hunk 1]
```diff
@@ -7,256 +7,287 @@
...
++    </param>
 ```
 
-## Potential downsides
 
-This feature introduces additional logics in both Core and Proxy. They may increase CPU usage on a head unit and mobile phone.
+## Potential downsides
 
-This feature requires Core and Proxies to buffer sent messages for a while. This will increase memory usage.
+- This feature introduces additional logics in both Core and Proxy and will increase their complexity.
```
Comment (theresalech) [inline proposals/0141-multiple-transports.md]: Please update to "logic"

File: proposals/0141-multiple-transports.md [hunk 2]
```diff
@@ -7,256 +7,287 @@
...
+- The authors have a use-case to prevent a video-streaming app from automatically launching on HMI if it is not connected with a high-bandwidth transport (i.e. Wi-Fi or USB). To realize this use-case, the resumption logic in SDL Core has to be modified. This feature will be addressed in another proposal [SDL-nnnn: Add capability to disable or reject an app based on app type, transport type and OS type][reg_limitation].
+- In some cases, SDL Core should not accept TCP connections on all network interfaces. For example, a head unit is equipped with a Wi-Fi network device and a communication module for cellar network, in which case the system will have two network interfaces. An OEM may want to accept SDL connection only through Wi-Fi's network interface and not through Internet connection. If we need to support such scenario, another proposal should be entered.
```
Comment (theresalech) [inline proposals/0141-multiple-transports.md]: Please update to "cellular network"

**Comment by shoamano83** [comment]
Thanks @theresalech , I've addressed your review comments.

**Comment by shoamano83** [comment]
Hi @theresalech, we had some internal discussions last week and I would like to bring the updated document to the discussion in SDLC. Could you kindly review the latest version again? If there are some grammar errors, I think I can fix them before the Steering Committee meeting.
Thanks!

File: proposals/0141-multiple-transports.md [hunk 3]
```diff
@@ -7,256 +7,282 @@
...
+* Instead of using Session ID as app identifier, introduce a new ID such as "App Instance ID" which is generated by Proxy and sent to Core during Version Negotiation. This approach should also work. The only downside is that we need to create a specification for the ID and Proxy may have additional complexity.
+* Support Hybrid service running on Secondary Transport. For this use-case, we will need a mechanism to seamlessly transfer the service between Primary and Secondary Transports. (Note that since Hybrid service is an extension of RPC service, it will be always started on Primary Transport.)
+* Extend the value of `secondaryTransport` parameter to accept multiple candidates of Secondary Transport. This will end up with Android Proxy running all three kinds of transports (Wi-Fi, Bluetooth and AOA) waiting for connections at the same time, and will introduces much complexity. If we need such advanced feature, perhaps another proposal can be entered in future, with a simplified implementation that Android Proxy will switch to another Secondary Transport only when current Secondary Transport is disconnected.
```
Comment (theresalech) [inline proposals/0141-multiple-transports.md]: Please update to "...and will introduce much complexity."  and "...can be entered in the future,"

**Comment by shoamano83** [comment]
Thank you very much @theresalech!

**Pull request merged**